### ISSUE TITLE
waitUntil test is failing 

### ISSUE THREAD

**Issue by echarles** [blois/js-module-renderer#2]
I have installed the js-module-renderer  in jupyterlab 2.1.0. All tests of the `tests.iypnb` runs fine except the `waitUntil` one.

<img width="749" alt="Screenshot 2020-05-11 at 08 00 40" src="https://user-images.githubusercontent.com/226720/81529039-47db8880-935e-11ea-9cf8-156a34a2ad3e.png">

**Comment by blois** [comment]
Yes, this test has the desired behavior, which is not implemented yet.
